test/acceptance/features/steps/steps.py

```diff
@@ -104,7 +104,10 @@ def given_db_operator_is_installed(context):
 @when(imported_nodejs_app_is_running_step)
 def imported_nodejs_app_is_running(context, application_name):
     namespace = context.namespace
-    application = NodeJSApp(application_name, namespace.name)
+    if application_name == 'nodejs-todo-git':
```

Comment:
This does not make much sense, why do you change a behavior of the based on the name? If the Node.js application requires a non-default image, that image should be parametrized in the step sentense itself and not in the implementation based on the name.

Reply:
step sentense? could you please tell me what it means.

Reply:
you mean here in feature file? `Imported Nodejs application "node-todo-git" is running`